pom.xml

```diff
@@ -15,8 +15,6 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
source and target version should be specified with one of the ways http://tutorials.jenkov.com/maven/java-compiler.html